apps/utils/locust_tests/lambda/server-load/README.md

```diff
@@ -0,0 +1,36 @@
+```mermaid
+sequenceDiagram
+    autonumber
+    actor u as User
+    participant sqs as SQS
+    participant b as Broker
+    participant c as Controller
+    participant n AS Worker<br />Nodes
+    participant asg as ASG
+    u -->> sqs: Start load test
+    activate sqs
+    sqs ->> b: Start Broker #lambda;
+    deactivate sqs
+    activate b
+    b ->> c: Launch Controller #lambda;
+    activate c
+    c -->> sqs: Report ready state and controller IP
+    activate sqs
+    sqs -->> b: Broker receives controller IP
+    deactivate sqs
+    loop Until scaling event
+        b ->> n: Launch Worker Node #lambda;
+        activate n
+        b ->> b: sleep 1s
+        Note right of b: Sleep interval should be a variable
+        b ->> sqs: Fetch messages in queue
+    end
+    break On ASG lifecycle hook "EC2_INSTANCE_LAUNCHING"
+        asg -->> sqs: Scaling event has ocurred
+        sqs -->> b: Broker receives scaling event
+        b ->> b: stop starting worker nodes
+        deactivate n
+    end
+    deactivate c
+    deactivate b
```

Comment:
For the first question, I just added a limit to the number of worker nodes that can be spawned, configurable via environment variable. (It's in `controller.py`) Right now it defaults to 80, which _should_ be more than enough.
Second question, everything gets torn down when we receive a scaling event notification. I'll add a timer after the signal so that it runs for a few seconds after just to get numbers after the scale. (Configurable via env var, naturally)